<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesignSuite</class>
 <widget class="QDialog" name="DesignSuite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PLMPanel</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#DesignSuite {
	margin-left: 5px;
	margin-right: 5px;
	margin-top: 5px;
	margin-bottom: 5px;
	border : 1.5px solid #232323;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="margin">
    <number>10</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0" rowspan="3">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>542</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>542</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/CLO_PLUGIN/INFOR/images/clo-govise-image.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="horizontalSpacing">
      <number>9</number>
     </property>
     <item row="0" column="2">
      <widget class="QToolButton" name="Cancel">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#Cancel {
    qproperty-icon: none;
    image: url(:/CLO_PLUGIN/INFOR/images/icon_cancel_none.svg);
}
#Cancel:hover {
	image: url(:/CLO_PLUGIN/INFOR/images/icon_cancel_over.svg);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">/*font: 75 16pt &quot;Times New Roman&quot;;*/
font: 75 12pt &quot;Tahoma&quot;;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Infor PLM Design Suite</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Tahoma&quot;;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Fabrics</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="ShMaterial">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color: #FFFFFF; background-color: #000000;&quot;&gt;Search&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/CLO_PLUGIN/INFOR/images/icon_search_over.svg);
/*#ShMaterial {
    qproperty-icon: none;
	image: url(:/CLO_PLUGIN/INFOR/images/icon_search_none.svg);
}
#ShMaterial:hover {
	image: url(:/CLO_PLUGIN/INFOR/images/icon_search_over.svg);
}*/</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Tahoma&quot;;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Trims</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="trimsSearch">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color: #FFFFFF; background-color: #000000;&quot;&gt;Search&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/CLO_PLUGIN/INFOR/images/icon_search_over.svg);
/*#trimsSearch {
    qproperty-icon: none;
	image: url(:/CLO_PLUGIN/INFOR/images/icon_search_none.svg);
}
#trimsSearch:hover {
	image: url(:/CLO_PLUGIN/INFOR/images/icon_search_over.svg);
}*/
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Tahoma&quot;;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Colors</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QToolButton" name="ShColor">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color: #FFFFFF; background-color: #000000;&quot;&gt;Search&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/CLO_PLUGIN/INFOR/images/icon_search_over.svg);
/*#ShColor {
    qproperty-icon: none;
	image: url(:/CLO_PLUGIN/INFOR/images/icon_search_none.svg);
}
#ShColor:hover {
	image: url(:/CLO_PLUGIN/INFOR/images/icon_search_over.svg);
}*/</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Tahoma&quot;;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Products</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QToolButton" name="searchProducts">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color: #FFFFFF; background-color: #000000;&quot;&gt;Search&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/CLO_PLUGIN/INFOR/images/icon_search_over.svg);
/*#searchProducts {
    qproperty-icon: none;
	image: url(:/CLO_PLUGIN/INFOR/images/icon_search_none.svg);
}
#searchProducts:hover {
	image: url(:/CLO_PLUGIN/INFOR/images/icon_search_over.svg);
}*/</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="4">
      <widget class="QToolButton" name="CreateProduct">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color: #FFFFFF; background-color: #000000;&quot;&gt;Create&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/CLO_PLUGIN/INFOR/images/icon_add_over.svg);
/*#CreateProduct {
    qproperty-icon: none;	
	image: url(:/CLO_PLUGIN/INFOR/images/icon_add_none.svg);
}
#CreateProduct:hover {
	image: url(:/CLO_PLUGIN/INFOR/images/icon_add_over.svg);
}*/</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="6">
      <widget class="QToolButton" name="UpdateProduct">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color: #FFFFFF; background-color: #000000;&quot;&gt;Update&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/CLO_PLUGIN/INFOR/images/icon_update_over.svg);
/*#UpdateProduct {
    qproperty-icon: none;	
	image: url(:/CLO_PLUGIN/INFOR/images/icon_update_none.svg);
}
#UpdateProduct:hover {
	image: url(:/CLO_PLUGIN/INFOR/images/icon_update_over.svg);
}*/</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="logout">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color: #FFFFFF; background-color: #000000;&quot;&gt;Logout&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/CLO_PLUGIN/INFOR/images/iconmonstr-log-out-11.svg);
/*#UpdateProduct {
    qproperty-icon: none;	
	image: url(:/CLO_PLUGIN/INFOR/images/icon_update_none.svg);
}
#UpdateProduct:hover {
	image: url(:/CLO_PLUGIN/INFOR/images/icon_update_over.svg);
}*/</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="styleSheet">
        <string notr="true">color: rgb(160, 160, 160);</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../PLMResource.qrc"/>
 </resources>
 <connections/>
</ui>
